### PR TITLE
[Issue #1548] sprint reports step function

### PR DESCRIPTION
**Pull request by coilysiren**
## Summary
Fixes https://github.com/HHS/simpler-grants-gov/issues/1548

## Code writing process

- renamed `copy_oracle_data.tf`
- copied `sfn_copy_oracle_data.tf` to `sfn_sprint_reports.tf`, renamed _"copy oracle data"_ inside of that file to _"sprint report"_
- updated the `Command` https://github.com/HHS/simpler-grants-gov/blob/4ee41df84d7a4504494b755ff900edcd41918e8b/infra/analytics/service/sfn_sprint_reports.tf#L28-L32
- added `aws_cloudwatch_log_group` to fix an error:

> Error: creating Step Functions State Machine (analytics-dev-sprint-reports): InvalidLoggingConfiguration: Invalid Logging Configuration: The CloudWatch Logs Resource Policy size was exceeded. We suggest prefixing your CloudWatch log group name with /aws/vendedlogs/states/.

## Changes proposed

Adds a sprint reporting data step function

## Testing

image

Not working yet due to https://github.com/HHS/simpler-grants-gov/issues/837

### CODE REVIEW

File: infra/analytics/service/sfn_sprint_reports.tf [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_log_group
+resource "aws_cloudwatch_log_group" "sprint_reports" {
+  name_prefix = "/aws/vendedlogs/states/${local.service_name}-sprint-reports"
+
+  # Conservatively retain logs for 5 years.
+  # Looser requirements may allow shorter retention periods
+  retention_in_days = 1827
+
+  # checkov:skip=CKV_AWS_158:skip requirement to encrypt with customer managed KMS key
+}
```
Comment (coilysiren) [inline infra/analytics/service/sfn_sprint_reports.tf:10]: We might want to backport this logs group pattern to the other step function. It fails to deploy periodically without this.

Comment (hhs-swift) [reply infra/analytics/service/sfn_sprint_reports.tf:10]: Sounds like a good idea! Is that something you want to do as part of this PR, or another one?

**Review by aplybeah — APPROVED** [review]

File: infra/analytics/service/sfn_sprint_reports.tf (hunk 1 above)
Comment (coilysiren) [reply infra/analytics/service/sfn_sprint_reports.tf:10]: https://github.com/HHS/simpler-grants-gov/pull/1615

Comment (coilysiren) [reply infra/analytics/service/sfn_sprint_reports.tf:10]: ^ New PR for the oracle SFN logs

**Pull request merged**